tools/packaging/packaging.psm1

```diff
@@ -2636,6 +2644,36 @@ function New-MSIPackage
     }
 }
 
+# generate a files.wxs for preview builds
+# so that the component ids are different than the stable builds
+# the file is created in the temp folder
+function New-PreviewFileWxs
+{
+    param
+    (
+        # File describing the MSI file components from the asset folder
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript( {Test-Path $_})]
+        [string] $FilesWxsPath = "$RepoRoot\assets\Files.wxs"
+    )
+
+    Write-Verbose "Generating new component Ids for Files-Preview.wxs" -Verbose
+    [xml] $filesAssetXml = Get-Content -Raw -Path $FilesWxsPath
+    foreach($component in $filesAssetXml.GetElementsByTagName('Component'))
+    {
+        $component.Id = $component.Id + "_Preview"
+    }
+
+    foreach($componentRef in $filesAssetXml.GetElementsByTagName('ComponentRef'))
+    {
+        $componentRef.Id = $componentRef.Id + "_Preview"
+    }
+
+    $previewFilesWxsPath = Join-Path ([System.IO.Path]::GetTempPath()) "Files-Preview.wxs"
```

Comment:
As far as I can see, this file does not get cleaned up at the end

Reply:
Good catch.  Will fix.